src/app/app.module.ts

```diff
@@ -32,8 +35,10 @@ import { SearchService } from "./search.service";
 import { UrlService } from "./url.service";
 import { CscComponent } from "./csc/csc.component";
 import { ProjectResourceComponent } from "./project-resource/project-resource.component";
-import { Ng2SmartTableModule } from "ng2-smart-table";
-import { TitleLinkComponent } from "./title-link/title-link.component";
+import { MatTableModule } from "@angular/material/table";
+import { MatSortModule } from "@angular/material/sort";
+import { MatCheckboxModule } from "@angular/material/checkbox";
```

Comment:
Is this used anywhere?

Reply:
Yes, in the csc and topics components. The TitleLinkComponent was not used anywhere.